snake.js

```diff
@@ -0,0 +1,6 @@
+class SnakeGame{
+    constructor(){}
+
+    render(){};
+    build(){};
```

Comment:
please swap `render` and `build` order to make it semantically correct - we first build game parts then render